circuits/cpp/barretenberg/cpp/src/barretenberg/stdlib/recursion/honk/verifier/ultra_recursive_verifier.cpp

```diff
@@ -7,50 +7,34 @@
 #include "barretenberg/numeric/bitop/get_msb.hpp"
 
 namespace proof_system::plonk::stdlib::recursion::honk {
-template <typename Flavor>
-UltraRecursiveVerifier_<Flavor>::UltraRecursiveVerifier_(Builder* builder,
-                                                         std::shared_ptr<typename Flavor::VerificationKey> verifier_key)
+
+template <typename Flavor, bool goblin_flag>
+UltraRecursiveVerifier_<Flavor, goblin_flag>::UltraRecursiveVerifier_(Builder* builder,
+                                                                      std::shared_ptr<VerificationKey> verifier_key)
     : key(verifier_key)
     , builder(builder)
 {}
 
-template <typename Flavor>
-UltraRecursiveVerifier_<Flavor>::UltraRecursiveVerifier_(UltraRecursiveVerifier_&& other) noexcept
-    : key(std::move(other.key))
-    , pcs_verification_key(std::move(other.pcs_verification_key))
-{}
-
-template <typename Flavor>
-UltraRecursiveVerifier_<Flavor>& UltraRecursiveVerifier_<Flavor>::operator=(UltraRecursiveVerifier_&& other) noexcept
```

Comment:
These are not currently in use and it's not clear they will be needed. I prefer to implement them and check their correctness as needed rather than leave them around to be used incorrectly.